data/transactions/logic/TEAL_opcodes.md

```diff
@@ -854,6 +854,17 @@ When A is a uint64, index 0 is the least significant bit. Setting bit 3 to 1 on
 - pop a byte-array A and integer B. Extract a range of bytes from A starting at B up to but not including B+8, convert bytes as big endian and push the uint64 result. If B+8 is larger than the array length, the program fails
 - LogicSigVersion >= 5
 
+## base64_decode e
+
+- Opcode: 0x5c {uint8 alphabet index}
+- Pops: *... stack*, []byte
+- Pushes: []byte
+- decode X which was base64-encoded using _encoding alphabet_ E. Fail if X is not base64 encoded with alphabet E
```

Comment:
I like the switch from A to E by going with encoding. Let's just avoid calling it an alphabet, just say "encoding".

Reply:
👍